AsyncDisplayKit/ASVideoNode.h

```diff
@@ -39,6 +39,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)play;
 - (void)pause;
 - (BOOL)isPlaying;
+- (void)reset;
```

Comment:
I like resetToPlaceholder, so I'll go with that unless anyone else has any ideas?